README.md

```diff
@@ -2,6 +2,7 @@
 
 [![GitHub Actions](https://github.com/messense/robotparser-rs/workflows/CI/badge.svg)](https://github.com/messense/robotparser-rs/actions?query=workflow%3ACI)
 [![Coverage Status](https://coveralls.io/repos/messense/robotparser-rs/badge.svg)](https://coveralls.io/r/messense/robotparser-rs)
+[![codecov](https://codecov.io/gh/Mephistophiles/robotparser-rs/branch/master/graph/badge.svg?token=WMBURQ0KQQ)](https://codecov.io/gh/Mephistophiles/robotparser-rs)
```

Comment:
This needs an update to point to `messense/robotparser-rs`

Reply:
Whoops, fixed
Interesting... Should I change the token?

Reply:
> token is not needed i think, just remove it 发自我的iPhone
> […](#)
> 在 2020年11月3日，18:51，Maxim Zhukov ***@***.***> 写道： ﻿ @Mephistophiles commented on this pull request. In README.md: > @@ -2,6 +2,7 @@ [![GitHub Actions]([https://github.com/messense/robotparser-rs/workflows/CI/badge.svg)](https://github.com/messense/robotparser-rs/actions?query=workflow%3ACI](https://github.com/messense/robotparser-rs/workflows/CI/badge.svg)%5D(https://github.com/messense/robotparser-rs/actions?query=workflow%3ACI)) [![Coverage Status]([https://coveralls.io/repos/messense/robotparser-rs/badge.svg)](https://coveralls.io/r/messense/robotparser-rs](https://coveralls.io/repos/messense/robotparser-rs/badge.svg)%5D(https://coveralls.io/r/messense/robotparser-rs)) +[![codecov]([https://codecov.io/gh/Mephistophiles/robotparser-rs/branch/master/graph/badge.svg?token=WMBURQ0KQQ)](https://codecov.io/gh/Mephistophiles/robotparser-rs](https://codecov.io/gh/Mephistophiles/robotparser-rs/branch/master/graph/badge.svg?token=WMBURQ0KQQ)%5D(https://codecov.io/gh/Mephistophiles/robotparser-rs)) Whoops, fixed Interesting... Should I change the token? — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.

done